perceval/backends/jira.py

```diff
@@ -163,6 +167,34 @@ def parse_issues(raw_page):
         for issue in issues:
             yield issue
 
+    @staticmethod
```

Comment:
I've been reading about the  [map()](https://docs.python.org/3/library/functions.html#map) function and seems it's exactly the same as:
```python
[f(x) for x in iterable]
```
So I've decided to go for that choice, also to get used with the list comprehensions. I've re-written the functions according to that approach:
```python
def map_custom_fields(custom_fields, issue):

    def complete_field(custom_fields, element):
        value = issue['fields'][element]
        issue['fields'][element] = {}
        issue['fields'][element]['value'] = value
        issue['fields'][element]['name'] = custom_fields[element]['name']
        issue['fields'][element]['id'] = custom_fields[element]['id']

    for element in issue['fields'].keys():
        [complete_field(custom_fields, element) for custom_field in custom_fields.keys() if custom_field == element]

    return issue
```

and 

```python
def filter_custom_fields(fields):

    custom_fields = {}

    _ = [field for field in fields if field['custom'] is True]

    for custom_field in _:
        custom_fields[custom_field['id']] = custom_field

    return custom_fields
```
What do you think?